src/tools/roc_recv/main.cpp

```diff
@@ -41,6 +43,15 @@ int main(int argc, char** argv) {
 
     pipeline::ReceiverConfig config;
 
+    if (args.list_drivers_given) {
+        core::HeapAllocator allocator;
```

Comment:
We already have HeapAllocator declaration in this function. Just move it to the top and use here.

Reply:
The same for roc_send version.